src/node-lib/auth/userManagement.ts

```diff
@@ -0,0 +1,10 @@
+/* istanbul ignore file */
+import { ManagementClient } from "auth0";
+
+import getServerConfig from "../getServerConfig";
+
+export const userManagementClient = new ManagementClient({
+  domain: getServerConfig("auth0IssuerBaseURL").replace("https://", ""),
+  clientId: getServerConfig("auth0UserManagementClientId"),
+  clientSecret: getServerConfig("auth0UserManagementClientSecret"),
+});
```

Comment:
This is the "OWA user management" application in Auth0. It only has access to update user's app metadata. 

Keeping this isolated will make it easier to extract the whole thing into a separate identity service at a later date.